code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1799,3 +1799,47 @@
 	else
 		M.adjustToxLoss(0.5*REAGENTS_EFFECT_MULTIPLIER)
 	. = 1
+
+/datum/reagent/medicine/limb_regrowth
+	name = "Carcinisoprojection Jelly"
+	description = "Also known as \"limb regrowth jelly\", this crabby looking xenobiological jelly will rapidly regrow any missing limbs someone has at the cost of some of their blood. Do not overdose."
+	taste_description = "salty slime"
+	color = "#EF5428" //like cooked crab!
+	reagent_state = LIQUID
+	overdose_threshold = 65 //it takes more than one bluespace syringe to overdose someone with this given how nasty the OD is.
+	value = REAGENT_VALUE_RARE
+
+/datum/reagent/medicine/limb_regrowth/reaction_mob(mob/living/carbon/C, method=TOUCH, reac_volume)
+	. = ..()
+	if(method == TOUCH) //as funny as it would be to have this hurled at someone, nah.
```

Comment:
Prevserve parent proc check (it returning false should prevent all further actions)
```suggestion
	. = ..()
	if(!.)
		return
	if(method == TOUCH) //as funny as it would be to have this hurled at someone, nah.
```